internal/serve/serve.go

```diff
@@ -163,6 +216,9 @@ func initHandlerDeps(cfg Configs) (handlerDeps, error) {
 		AccountSponsorshipService: accountSponsorshipService,
 		PaymentService:            paymentService,
 		AppTracker:                cfg.AppTracker,
+		// TSS
+		RPCCallerServiceChannel: rpcCallerServiceChannel,
+		TSSRouter:               router,
```

Comment:
Which of the two will be the entrypoint when handling HTTP requests? Probably that one can be our only dependency to be injected in `handlerDeps`, right?

Reply:
We actually need both, because the channel is being closed (via Stop()) in the Serve function

Reply:
But the entrypoint will be the router

Reply:
Hmm got it, maybe we can create an `OnStop` function on the router for this?